operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -337,7 +341,7 @@ public static Step bringAdminServerUp(DomainPresenceInfo info, PodAwaiterStepFac
 
   @Override
   public void runMakeRight(MakeRightDomainOperation operation) {
-    final DomainPresenceInfo liveInfo = operation.getPresenceInfo();
+    final DomainPresenceInfo liveInfo = (DomainPresenceInfo) operation.getPresenceInfo();
```

Comment:
Consider adding generics to MakeRightOperation to remove the need of this cast. Please see suggested changes at https://github.com/oracle/weblogic-kubernetes-operator/commit/54122bb80e88a19f162cfc249871438de03fce3b

Reply:
Good suggestion. will do.

Reply:
done